api/users/serializers.py

```diff
@@ -117,6 +123,25 @@ def get_absolute_url(self, obj):
             'version': self.context['request'].parser_context['kwargs']['version']
         })
 
+    def get_node_count(self, obj):
+        return default_node_list_queryset(model_cls=Node).filter(contributor__user__id=obj.id).count()
+
+    def get_quickfiles_count(self, obj):
+        return QuickFilesNode.objects.get(contributor__user__id=obj.id).files.filter(type='osf.osfstoragefile').count()
```

Comment:
It doesn't look like we're permission checking on QuickFilesNodes so it looks like everyone can see everyone's quickfiles, right?  So this query looks good to me.

Reply:
Yeah quickfiles are all immutably public as far as I know